lib/pal/InformationProviderImpl.cpp

```diff
@@ -40,11 +40,17 @@ namespace PAL_NS_BEGIN {
 
     void InformatonProviderImpl::UnRegisterInformationChangedCallback(int callbackToken)
     {
+        // The callbackToken cannot be less than 1.
+        if (callbackToken < 1)
+        {
+            return;
+        }
+
         size_t index = (size_t)(callbackToken - 1);
```

Comment:
Can we do `static_cast<size_t>` instead of raw cast?

Reply:
Yes. I was avoiding touching the code to keep things small, but can move to a static cast.